airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/ToEmit.java

```diff
@@ -49,9 +49,9 @@ public enum ToEmit {
       MetricClientFactory.getMetricClient().distribution(OssMetricsRegistry.NUM_ACTIVE_CONN_PER_WORKSPACE, count);
     }
   })),
-  NUM_ABNORMAL_SCHEDULED_SYNCS(countMetricEmission(() -> {
-    final var count = ReporterApp.configDatabase.query(MetricQueries::numOfJobsNotRunningOnSchedule);
-    MetricClientFactory.getMetricClient().gauge(OssMetricsRegistry.NUM_ABNORMAL_SCHEDULED_SYNCS, count);
+  PCT_ABNORMAL_SCHEDULED_SYNCS(countMetricEmission(() -> {
```

Comment:
to make sure I understand, this metric is computed over the last 24 hours and emitted every hour?

Reply:
If a connection was scheduled to run X hours, we pull the last 24 hours. If a connection was scheduled to run every X minutes, we only pull the last 1 hour. (concerning we might pulling too many job records if extends to 24 hours; but in reality most jobs are scheduled to be running on hours so i think it should be fine to extend to 24 hours.) And yes we only send out the metric once an hour.